fsw/cfe-core/src/sb/cfe_sb_priv.h

```diff
@@ -413,18 +413,16 @@ int32 CFE_SB_RemoveDest(CFE_SB_RouteEntry_t *RouteEntry, CFE_SB_DestinationD_t *
 **
 ** \par Assumptions, External Events, and Notes:
 **          - For statically defined messages, a function call will not work.  The 
-**            macros #CFE_SB_CMD_HDR_SIZE and #CFE_SB_TLM_HDR_SIZE are available for use 
+**            macros cfe_sb.h::CFE_SB_CMD_HDR_SIZE and cfe_sb.h::CFE_SB_TLM_HDR_SIZE are available for use 
```

Comment:
Do these link correctly?

Reply:
These are not working, I missed them too. Will fix